### DISCUSSION TITLE
I added a custom metrics on the sw-ui. Its scope is endponit, but there is no request for the metrics data

### DISCUSSION THREAD

**Discussion by Alberter** [apache/skywalking#7497] [Q&A] [answered]
step:
1.The oal expression is: endpoint_abnormal = from(Endpoint.*).filter(responseCode in [404, 500, 503]).count()
2.sw-ui

![image](https://user-images.githubusercontent.com/29748672/130061760-2b16c456-f2ee-4d06-9a23-05140958aa92.png)
3.As described above , there is no request for the metrics data which my metrics

**Comment by kezhenxu94** [comment]
@Alberter you need to configure the UI to display your metrics, by default it only display our built-in metrics, you can click the lock icon <img width="37" alt="Screen Shot 2021-08-19 at 20 22 32" src="https://user-images.githubusercontent.com/15965696/130067584-135d7fac-28df-4579-b52e-a80430d4f3d5.png"> to add a panel to show your metrics.

And make sure to set the "Chart Type" as `Digital Card` because your metrics type is count

**Reply by Alberter** [reply]
![image](https://user-images.githubusercontent.com/29748672/130071153-b6ee9fe2-90c3-415c-aad9-7cabd62f3973.png)
this is my configure on the UI

**Reply by Alberter** [reply]
Modifying an existing  metrics is successfully displayed,like this
![image](https://user-images.githubusercontent.com/29748672/130071828-173f6427-7eca-4431-a588-314d2a918d84.png)
![image](https://user-images.githubusercontent.com/29748672/130071866-831ac98d-2662-4111-a6a9-7f43a26c2e94.png)
Adding a panel to show my metrics failed

**Reply by wu-sheng** [reply]
Do you notice the difference of your 2 screenshots? Your one has `selector` option, I think? Could you use EN mode?

**Reply by wu-sheng** [reply]
What kind of dashboard are you creating? I can't see that option in the demo env.

**Reply by Alberter** [reply]
sorry There is a problem with that screenshot. I just want to create the EN module
![image](https://user-images.githubusercontent.com/29748672/130074986-96251013-582b-4863-8f8b-9fb86f4dbd0b.png)
![image](https://user-images.githubusercontent.com/29748672/130075089-9e2f3b9c-7d70-4b28-a9f2-47825dde314b.png)
The above picture configuration display failed, but the entity type can be successfully displayed by changing to "all",like this
![image](https://user-images.githubusercontent.com/29748672/130076465-fee7ab96-93f8-4c84-9caa-c4911610add1.png)

**Reply by wu-sheng** [reply]
My point is this

![image](https://user-images.githubusercontent.com/5441976/130077025-eaa21cc9-d354-4710-9a93-fe82a93bcbb6.png)

**Reply by wu-sheng** [reply]
I don't know how you make this showing up. Whether is this a bug or anything special in your browser. I can't see this on my side.

**Reply by wu-sheng** [reply]
I only can see this in the NOC dashboard

![image](https://user-images.githubusercontent.com/5441976/130077768-52eb7afd-68fb-42fc-9164-ff35abd1b9dc.png)

**Comment by wu-sheng** [comment]
I think you should show up how you configure on the UI. You missed the key point.

**Comment by wu-sheng** [comment, marked as answer]
One more thing, which version are you using? I think there was a version(8.5.0 or 8.6.0) having UI side setting up issue.

**Reply by Alberter** [reply]
My version is 8.6.0

**Reply by wu-sheng** [reply]
I don't remember which version the bug belongs, but it is definitely there. So, check our demo env, if it is not as same as your env, then it was a fixed bug.

**Reply by Alberter** [reply]
ok thanks
